src/util/errorUtil.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from '../util';
+
+export class ErrorData {
```

Comment:
How about exposing ErrorData via ErrorData.ts and having a constructor that takes an error/any?

Reply:
Makes complete sense. Resolved